### PR TITLE
Update proptypes for Card row

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR patches a few PropTypes issues with the CardRow for a quieter console.

## Testing in `kajabi-products`
1. (LOW) Patch to proptypes issue in CardRow. No impact on existing uss.

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
LGTM! 👍🏼

File: packages/sage-react/lib/Card/CardRow.jsx [hunk 1]
```diff
@@ -32,18 +32,21 @@ export const CardRow = ({
   );
 };
 
+CardRow.GAP_OPTIONS = CARD_ROW_GAP_OPTIONS;
```
Comment (monkeypox8) [inline packages/sage-react/lib/Card/CardRow.jsx:35]: I was looking at the config and both `START` and `TOP` settings are assigned to `start`, which got me curious. Any idea if this was by intention?

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM! Left one non-blocking question in the config.

File: packages/sage-react/lib/Card/CardRow.jsx (hunk 1 above)
Comment (philschanely) [reply packages/sage-react/lib/Card/CardRow.jsx:35]: Shoot I just missed this before merging. Will look closer but yes I do believe it is. Idea is to do both a "CSS" thinker's approach and a more developer-focused. AKA if a dev wants to align do they think of that as "top" or the css property of "start"? Providing both here.

**Pull request merged**